### DISCUSSION TITLE
Come cambiare i colori di default della barra di stato?

### DISCUSSION THREAD

**Discussion by pigreco** [ondata/guidaVisiData#25] [Domande] [answered]
Con la nuova versione `saul.pw/VisiData v2.6.1` VisiData si presetna così:

![image](https://user-images.githubusercontent.com/7631137/140392867-981147b1-3e5f-489a-ba00-9137c2ffd34d.png)

il colore della barra di stato non mi permette di visualizzare bene i vari messaggi e vorrei cambiarlo o ottimizzare la visualizzazione.

Digitando `Shift + o` raggiungo le opzioni:

![image](https://user-images.githubusercontent.com/7631137/140393313-88483cb8-c82b-4db8-b728-8803fc73b1f2.png)
 ma come cambio il valore evidenziato dalla freccia verde?

grazie mille

**Comment by aborruso** [comment, marked as answer]
Ciao @pigreco ,
se vai in quella riga, e in quella colonna, e fai click su `e`, puoi scrivere ad esempio `red` e dare invio. E vedrai che cambia.
Ma vale soltanto per questa sessione.

Se vuoi che diventi globale, devi seguire le istruzione citate [qui](https://www.visidata.org/docs/customize/):

- modifichi il file di config che sta nella home dell'utente ` ~/.visidatarc`
- e aggiungi l'opzione relativa che vuoi cambiare con questa sintassi di esempio `options.color_status = 'bold black on 255'`, e salvi quel file.

Così è diventata globale e sarà sempre valida

Nella guida, si parla di questi temi qui https://ondata.github.io/guidaVisiData/testo/#riga-attiva-di-colore-nero
